### PR TITLE
create and style educator profile screen

### PR DESCRIPTION
**Pull request by crathor**

### CODE REVIEW

File: js/screens/Educator/Educator.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import call from 'react-native-phone-call';
+
+const EducatorContact = ({ user }) => (
+  <View style={styles.educatorInfoContainer}>
+    <Text>{user.name}</Text>
+    <Text>{user.phone}</Text>
+  </View>
+);
+const Educator = ({ user }) => {
+  console.log(user);
```
Comment (siddharthparmar7) [inline js/screens/Educator/Educator.js]: Please remove this console log.

File: js/screens/Educator/EducatorContainer.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  }
+`;
+
+export default class EducatorContainer extends Component {
+  static navigationOptions = {
+    title: 'Profile',
+  };
+  render() {
+    return (
+      <Query query={GET_USER} variables={{ id: 'cjm1jvtfi04qy0197ww436yqq' }}>
```
Comment (siddharthparmar7) [inline js/screens/Educator/EducatorContainer.js]: hardcoded? is this temporary?

File: js/screens/Educator/Educator.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        <Text style={styles.infoText}>Mon-Fri 3pm-6pm. Call to Confirm</Text>
+      </View>
+      <View style={styles.buttonContainer}>
+        <TouchableOpacity
+          style={styles.contactButton}
+          onPress={() =>
+            call({
+              number: user.phone.replace(/(\s+)/g, ''),
+              prompt: true,
+            }).catch(console.error)
```
Comment (siddharthparmar7) [inline js/screens/Educator/Educator.js]: instead of consoling an error, just return it.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please use activity indicator show loading, instead of the frog we are using.

File: js/screens/Educator/EducatorContainer.js (hunk 2 above)
Comment (crathor) [reply js/screens/Educator/EducatorContainer.js]: yes this is temporary

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**